api/src/integration/java/org/pmiops/workbench/google/DirectoryServiceImplIntegrationTest.java

```diff
@@ -47,8 +47,14 @@ public void testDirectoryServiceUsernameIsTaken() {
   @Test
   public void testCreateAndDeleteTestUser() {
     String userName = String.format("integration.test.%d", Clock.systemUTC().millis());
-    service.createUser("Integration", "Test", userName, "notasecret");
+    service.createUser("Integration", "Test", userName, "notasecret@gmail.com");
```

Comment:
Interestingly, now that we're storing a user's contact email address in a custom schema field with a typed value of "email address", this test was failing (API response 400 from Google) with the original "notasecret" value. Yay for extra field validation!